src/extension/tezos-types.ts

```diff
@@ -0,0 +1,144 @@
+import { TezosWrappedOperation } from '@airgap/coinlib-core'
+
+export type MetadataBalanceUpdatesKindEnum = 'contract' | 'freezer'
```

Comment:
Do we need all those types? Are they not in the coinlib?

Reply:
I reduced the types to the minimum necessary